src/app/collective-rewards/rewards/builders/EstimatedRewards.tsx

```diff
@@ -104,7 +104,15 @@ export const EstimatedRewards: FC<EstimatedRewardsProps> = ({ tokens: { rif, rbt
         title="Estimated rewards"
         data-testid="EstimatedRewards"
         tooltip={{
-          text: 'The information displayed is dynamic and may vary based on total rewards available and user activity. This data is provided for informational purposes only. Please note that the final reward amount will be determined at the end of the cycle.',
+          text: (
+            <p>
```

Comment:
@antomor this is causing the following error:

```
hook.js:608 Warning: In HTML, <p> cannot be a descendant of <p>.
This will cause a hydration error. Error Component Stack
```

Reply:
Fixed it